packages/directory/addon/components/dir-table.ts

```diff
@@ -72,7 +76,7 @@ export default class DirTableComponent extends Component<DirTableComponentArgs>
         classNames: 'dir-table__header-cell dir-table__header-cell--actions',
         cellComponent: 'dir-asset-row-actions',
         cellClassNames: 'dir-table__cell dir-table__cell--actions',
-        breakpoints: ['tablet', 'desktop', 'jumbo'],
+        breakpoints: ['mobile', 'tablet', 'desktop', 'jumbo'],
```

Comment:
Does this look ok on mobile?

Reply:
lol nope! That was supposed to be on the favorite cell, not the action cell. Good catch